frontend/src/app/esri-map/esri-map.component.ts

```diff
@@ -59,10 +58,32 @@ export class EsriMapComponent implements AfterViewInit {
         });
 
         function checkForest (feature) {
+          let fwfRoute;
+          let fwf;
           const mbsForest = 'Mt. Baker-Snoqualmie National Forest';
           const common = feature.graphic.attributes.COMMONNAME;
           const route = document.location.origin;
-          if (common === mbsForest) {
+          const fuelwoodForest = [
+            {fname: 'Idaho Panhandle National Forests', fshort: 'ipnf'},
+            {fname: 'Flathead National Forest', fshort: 'flathead'},
+            {fname: 'Chattahoochee-Oconee National Forest', fshort: 'conf'},
+            {fname: 'George Washington & Jefferson National Forest', fshort: 'gwj'},
+            {fname: 'Shoshone National Forest', fshort: 'shoshone'},
+            {fname: 'Hiawatha National Forest', fshort: 'hiawatha'},
+            {fname: 'Grand Mesa Uncompahgre & Gunnison National Forest', fshort: 'gmug'},
+          ];
+          for (let i = 0; i < fuelwoodForest.length; i++) {
+            if (common === fuelwoodForest[i].fname) {
+              fwf = fuelwoodForest[i].fname;
+              fwfRoute = fuelwoodForest[i].fshort;
+            }
+          }
```

Comment:
Are y'all using any polyfills? I'm not sure if TypeScript has them baked in or not, but if so, what do you think of replacing this with a destructuring assignment and an `Array.find`:

```suggestion
const { fname: fwf, fshort: fwRoute } = fuelwoodForest.find(forest => common === forest.name);
```
```

Reply:
@mgwalker it looks like we have a PR open to merge `dev` into `staging`, I think that's why we're seeing some stuff not directly related to 153 popping up here. So perhaps I jumped the gun on opening up this PR 😬  ... I was just hoping to be able to test the paygov integration which has to happen on staging.

I'd like to address the stuff you found in your review but I'm not sure if it makes sense to do that in this PR, or the other one. Whatcha guys think? @mtlaney @mgwalker

Reply:
For convenience [here](https://github.com/USDAForestService/fs-open-forest-platform/pull/1449) is the PR for `dev` -> `staging`

Reply:
Probably the other one, if it's more related to that. I'd also be cool with punting to a later PR dedicated solely to dealing with these couple of small things if that's easier. Maybe we can create an issue tagged as a small cleanup, just to make sure we don't forget about it, in that case.

Reply:
I'd be good cleaning this up at a later time, it was coded out of frustration with ESRI hating any modern fat arrow functions or refusing to take parameters in the checkForest function

Reply:
Alright great, I like that option as well. I'm going to go ahead and merge this in then so I can keep working on it.